app/controllers/home_controller.rb

```diff
@@ -1,7 +1,6 @@
 class HomeController < ApplicationController
-
   def index
-    @neighborhoods = NeighborhoodAnalyst.top_three
+    service = GoogleService.new
```

Comment:
Depending if we want to add more actions using this service, might we want to move this down to a private memoization method? Everything else looks great!